### ISSUE TITLE
YouTube broke NewPipe again...

### ISSUE THREAD

**Issue by J-Man951** [TeamNewPipe/NewPipe#7842]
I received an update and videos were playing again for a day or two before getting g the same result as before. I hope this gets fixed soon! This is a great app!

**Comment by TiA4f8R** [comment]
Could you:

- make sure (but **really**) you are using the 0.21.16 version;
- explain more your issue (with a crash report if possible);
- follow our bug report template,

please? Thank you.

**Comment by TheBiostarfly** [comment]
## Exception
* __User Action:__ requested stream
* __Request:__ https://www.youtube.com/watch?v=SqCf1S18M0
* __Content Country:__ DE
* __Content Language:__ de-DE
* __App Language:__ de_DE
* __Service:__ YouTube
* __Version:__ 0.21.16
* __OS:__ Linux Android 12 - 31
<details><summary><b>Crash log </b></summary><p>

```
org.schabi.newpipe.extractor.stream.StreamInfo$StreamExtractException: Could not get any stream. See error variable to get further details.
	at org.schabi.newpipe.extractor.stream.StreamInfo.extractStreams(StreamInfo.java:194)
	at org.schabi.newpipe.extractor.stream.StreamInfo.getInfo(StreamInfo.java:72)
	at org.schabi.newpipe.extractor.stream.StreamInfo.getInfo(StreamInfo.java:64)
	at org.schabi.newpipe.util.ExtractorHelper.lambda$getStreamInfo$3(ExtractorHelper.java:116)
	at org.schabi.newpipe.util.ExtractorHelper.$r8$lambda$YTHJjScxCJNO1LTCqs3IKy35iyY(Unknown Source:0)
	at org.schabi.newpipe.util.ExtractorHelper$$ExternalSyntheticLambda6.call(Unknown Source:4)
	at io.reactivex.rxjava3.internal.operators.single.SingleFromCallable.subscribeActual(SingleFromCallable.java:43)
	at io.reactivex.rxjava3.core.Single.subscribe(Single.java:4813)
	at io.reactivex.rxjava3.internal.operators.single.SingleDoOnSuccess.subscribeActual(SingleDoOnSuccess.java:35)
	at io.reactivex.rxjava3.core.Single.subscribe(Single.java:4813)
	at io.reactivex.rxjava3.internal.operators.single.SingleSubscribeOn$SubscribeOnObserver.run(SingleSubscribeOn.java:89)
	at io.reactivex.rxjava3.core.Scheduler$DisposeTask.run(Scheduler.java:614)
	at io.reactivex.rxjava3.internal.schedulers.ScheduledRunnable.run(ScheduledRunnable.java:65)
	at io.reactivex.rxjava3.internal.schedulers.ScheduledRunnable.call(ScheduledRunnable.java:56)
	at java.util.concurrent.FutureTask.run(FutureTask.java:266)
	at java.util.concurrent.ScheduledThreadPoolExecutor$ScheduledFutureTask.run(ScheduledThreadPoolExecutor.java:301)
	at java.util.concurrent.ThreadPoolExecutor.runWorker(ThreadPoolExecutor.java:1167)
	at java.util.concurrent.ThreadPoolExecutor$Worker.run(ThreadPoolExecutor.java:641)
	at java.lang.Thread.run(Thread.java:920)

```
</details>
<hr>

I think it could be the same issue im having

**Comment by J-Man951** [comment]
Thank you for getting back to me so quickly! I don't know what changed... I do have the current version installed and when I checked a few minutes ago it was working again... I am also using the TrackerControl app from F-Droid on both my mobile devices. I've found that the only way to get NewPipe to work with TrackerControl is to disable monitoring and blocking... I hope this helps you out. Either way, it's working now. Sorry for the trouble and thank you for the App!
